keyboards/pimoroni/keybow2040/mcuconf.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+/*
+ * implicitly LED driver chips
+ */
+#define DRIVER_COUNT 1
+
+/*
+ * I2C address when IS31FL3731 AD pin is connected to ground
+ */
+#define DRIVER_ADDR_1 0b1110100
+
+#undef  RP_I2C_USE_I2C0
+#undef  RP_I2C_USE_I2C1
+#define RP_I2C_USE_I2C0 TRUE
+#define RP_I2C_USE_I2C1 FALSE
+
+#undef  I2C_DRIVER
+#define I2C_DRIVER I2CD0
+
+#define I2C0_SDA_PIN GP4
+#define I2C0_SCL_PIN GP5
+#define I2C0_CLOCK_SPEED 100000
+
+/*
+ * I2C1 should be unused, but there seem to be parts of the driver
+ * that assume I2C1 is used and I2C0 is not, so tell both of them to
+ * use the same GPIO pins. This is necessary for it to build and work
+ * correctly.
+ */
+#define I2C1_SDA_PIN GP4
+#define I2C1_SCL_PIN GP5
+#define I2C1_CLOCK_SPEED 100000
```

Comment:
These parameters should also be set in `config.h`, not here.
```suggestion
```

Reply:
Fixed in round three!